windows-gaming-driver/src/driver/control/codec.rs

```diff
@@ -1,11 +1,16 @@
 use std::io;
 use std::str;
-use bytes::BytesMut;
+use bytes::{BytesMut, BufMut, LittleEndian, IntoBuf, Buf};
 use tokio_io::codec::{Encoder, Decoder};
 
 #[derive(Debug, PartialEq, Eq)]
 pub enum ControlCmdOut {
-    // none yet
+    MouseEdged {
+        x: i32,
+        y: i32,
+    },
+    TemporaryLightAttach,
+    TemporaryLightDetach,
```

Comment:
These appear to be responses. The name has to reflect that: TemporaryLightAttach*ed*